rsconnect/metadata.py

```diff
@@ -721,7 +731,7 @@ def update_content_item_last_build_time(self, guid: str, defer_save: bool = Fals
         """
         with self._lock:
             content = self.get_content_item(guid)
-            content["rsconnect_last_build_time"] = datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ")
+            content["rsconnect_last_build_time"] = datetime.now(timezone.utc).strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
Note that this change will not result in changed output, because strftime() is being called with a format string.